### PR TITLE
Display asset filesize + link to asset metadata

### PR DESCRIPTION
**Pull request by mvandenburgh**
Display asset filesize and link to asset metadata in the file browser.

### CODE REVIEW

**Comment by AlmightyYakob** [comment]
Will take a look once I get my dev environment set up again (new laptop).

File: web/src/views/FileBrowserView/PublishFileBrowser.vue [hunk 1]
```diff
@@ -123,6 +126,25 @@
                   </v-icon>
                 </v-btn>
               </v-list-item-action>
+
+              <v-list-item-action v-if="item.asset_id">
+                <v-btn
+                  icon
+                  :href="assetMetadataURI(item.asset_id)"
```
Comment (satra) [inline web/src/views/FileBrowserView/PublishFileBrowser.vue:133]: can this href target a new window?

Comment (AlmightyYakob) [reply web/src/views/FileBrowserView/PublishFileBrowser.vue:133]:
```suggestion
                  :href="assetMetadataURI(item.asset_id)"
                  target="_blank"
                  rel="noopener"
```

**Review by waxlamp — APPROVED** [review]
Looks very nice.

**Pull request merged**